freshmaker/errata.py

```diff
@@ -470,3 +474,18 @@ def get_cve_affected_rpm_nvrs(self, errata_id):
                                     nvrs.update(just_nvrs)
 
         return list(nvrs)
+
+    def get_blocking_advisories_builds(self, errata_id):
+        """ Get all advisories that block given advisory id, and fetch all builds from it
+
+        :param number errata_id: ID of advisory
```

Comment:
```suggestion
        :param int errata_id: ID of advisory
```

Reply:
Well actually I am not sure about `int` because errata endpoint returns list of strings like this "2020:3095" and it's not an int but probably it could be, not sure about the return format.

Reply:
Oh yeah, good point. Maybe omit the type?

Reply:
Is it legal? =))